docs/DeveloperGuide.adoc

```diff
@@ -315,16 +330,16 @@ private String[] getRangeFromInput() {
     }
 ----
 
-To determine which contacts should be added to the exportBook we have to check the the user input, there are 3 cases:
+To determine which contacts should be added to the `exportBook` we have to check the the user input, there are three cases:
```

Comment:
double "the"
I think we turn the comma into a full stop and start a new sentence:
To determine which contacts should be added to the `exportBook` we have to check the user input. There are three cases: